.github/workflows/index-update.yml

```diff
@@ -0,0 +1,165 @@
+              exit 1
+            fi
+            # Check repo can be loaded
+            if ! helm repo add cache file://../download/ ; then
+              echo "New index.yaml file can't be used as a file"
+              exit 1
+            else
+              # Remove the repo
+              helm repo remove cache
+            fi
+            cp ../download/index.yaml bitnami/index.yaml
+
+            ## Build and push promotions
+            tar -xzf ../download/${chart_name}-${chart_version}.tgz -C ../download
+            image_list="[]"
+            # Get image list (removing the registry)
+            for chart_yaml in $(find "../download/${chart_name}" -name "Chart.yaml"); do
+              image_list_aux="$(yq '.annotations.images' "${chart_yaml}" | yq '[ .[] | .image | sub("^[^/]+/", "") ] | tojson')"
+              image_list="$(jq -c --null-input --argjson arr1 "$image_list" --argjson arr2 "$image_list_aux" '$arr1 + $arr2 | unique')"
+            done
+            app_version=$(yq '.appVersion' "../download/${chart_name}/Chart.yaml")
+            release_date="$(date -u +"%Y/%m/%d")"
+            file_prefix="$(date -u +"%s%3N-${chart_name}-${app_version}")"
+            # Build JSON files
+            jq --null-input \
+              --arg platform_id "bitnami-chart-debian-x64" \
+              --arg chart_name "${chart_name}" \
+              --arg app_version "${app_version}" \
+              --arg chart_version "${chart_version}" \
+              --arg bundled_os_version "12" \
+              --argjson image_list "${image_list}" "${promotion_json_template}" > "${file_prefix}-bitnami-chart-debian-x64.json"
+            jq --null-input \
+              --arg platform_id "vmware-chart-debian-x64" \
+              --arg chart_name "${chart_name}" \
+              --arg app_version "${app_version}" \
+              --arg chart_version "${chart_version}" \
+              --arg bundled_os_version "12" \
+              --argjson image_list "${image_list}" "${promotion_json_template}" | jq '.properties += {"alias_platform_from": "bitnami-chart-debian-x64"}' > "${file_prefix}-vmware-chart-debian-x64.json"
+            # Upload files to the release bucket
+            aws s3 cp "${file_prefix}-bitnami-chart-debian-x64.json" "s3://${AWS_PROMOTION_BUCKET}/releases/${release_date}/${file_prefix}-bitnami-chart-debian-x64.json"
+            aws s3 cp "${file_prefix}-vmware-chart-debian-x64.json" "s3://${AWS_PROMOTION_BUCKET}/releases/${release_date}/${file_prefix}-vmware-chart-debian-x64.json"
+
+            # Remove chart files
+            rm -rf ../download
+          done
+
+          # Avoid overriding index branch when remote commit does not match our checkout commit
+          current_commit_id=$(git rev-parse index)
+
+          # Push changes
+          git add bitnami/index.yaml && git commit --signoff --amend --no-edit
+          git push origin index --force-with-lease=index:${current_commit_id}
```

Comment:
> This push will be done by Bitnami Bot and it could trigger other workflows. Is this expected?

Yes it is expected -> https://github.com/bitnami/charts/blob/index/.github/workflows/archive-full-index.yml

Reply:
Yes, it is expected and intended. This logic is inspired by the current 'index-update' logic, but instead of using VIB publish as trigger, it will run on a schedule and use the OCI registry as source of truth.